main.go

```diff
@@ -69,7 +69,7 @@ func main() {
 	flag.StringVar(&opts.delay, "p", "", "Seconds of `delay` between requests, or a range of random delay. For example \"0.1\" or \"0.1-2.0\"")
 	flag.StringVar(&opts.filterStatus, "fc", "", "Filter HTTP status codes from response. Comma separated list of codes and ranges")
 	flag.StringVar(&opts.filterSize, "fs", "", "Filter HTTP response size. Comma separated list of sizes and ranges")
-	flag.StringVar(&opts.filterRegexp, "fr", "", "Filter regexp")
+	flag.StringVar(&opts.filterRegexp, "fr", "", "Filter regexp, also fuzz keyword allowed")
```

Comment:
I don't think this needs a mention in the help text, as I think the default assumption here is that everything works with the keywords ;)

Reply:
Removed.